### DISCUSSION TITLE
I've created a chroot-based container tool, I think it could be useful for using as a helper to this project

### DISCUSSION THREAD

**Discussion by takusuman** [dslm4515/CMLFS#56] [Show and tell]
When I was initially playing with Musl-LFS, I always had a problem with the chroot stage: I had to rely on my `.*sh_history` when exporting the variables to the chroot environment (y'know, using `/tools/bin/env -i` ...), that gave me the idea to create a tool that, besides being a chroot-based containerizer (analogue to Solaris Zones or Jails), also could automatize this process.
The idea remained dormant until September, when I finally polished a small hack that I was using and transformed in a real script.
It's called Mitzune, it's fully written in Shell script (GNU dialect for now) and relies only on basic UNIX tools, tar, xz and gzip.
Here's a link to it: https://github.com/Projeto-Pindorama/mitzune

Currently, I'm using it in my project, but I thought that it may would be useful for the rest of you to use.

What do you guys think about it?
This is not related to Musl-LFS/CMLFS at all, but anyway.

**Comment by dslm4515** [comment]
The few times I built MLFS under another distro (like Ubuntu) I can never clear the environmental variables completely. For example in Ubuntu, I create a new user with my own prompt (the $PS1) but when I login, I still get the Ubuntu customized $PS1.

**Reply by takusuman** [reply]
I use Arch and I never saw this happening here. May it's because Ubuntu customizations to GNU Bourne-Again Shell?

note: sorry, I posted instead of replying

**Reply by dslm4515** [reply]
> I use Arch and I never saw this happening here. May it's because Ubuntu customizations to GNU Bourne-Again Shell?

I am not sure. I could never find the source builds for Ubuntu...unlike Void and Alpine that have it hosted on GitHub.

_Once I had a laptop that would hang on reboots when I booted my own kernel. But if I booted a kernel from Ubuntu, issue gone. I could never find what patches Ubuntu used._
